webpack.config.js

```diff
@@ -42,6 +50,7 @@ module.exports = {
             options: {
               importLoaders: 1,
               modules: true,
+              esModule: false,
             },
           },
```

Comment:
Need to add `esModule: false` option to upgraded `css-loader`
https://github.com/vuejs/vue-style-loader/issues/46#issuecomment-670624576